### PR TITLE
Update helm readme and docs

### PR DESCRIPTION
**Pull request by vepatel**
### Proposed changes

- Update helm docs with sections and values overwritten by release script

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that all unit tests pass after adding my changes
- [ ] I have updated necessary documentation
- [x] I have rebased my branch onto main
- [x] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

**Review by jjngx — APPROVED** [review]

File: docs/content/installation/installation-with-helm.md [hunk 1]
```diff
@@ -338,10 +466,17 @@ The following tables lists the configurable parameters of the NGINX Ingress Cont
 |`prometheus.port` | Configures the port to scrape the metrics. | 9113 |
 |`prometheus.scheme` | Configures the HTTP scheme to use for connections to the Prometheus endpoint. | http |
 |`prometheus.secret` | The namespace / name of a Kubernetes TLS Secret. If specified, this secret is used to secure the Prometheus endpoint with TLS connections. | "" |
+|`prometheus.service.create` | Create a Headless service to expose prometheus metrics. Requires `prometheus.create`. | false |
+|`prometheus.service.labels` | Kubernetes object labels to attach to the service object. | "" |
```
Comment (shaun-nx) [inline docs/content/installation/installation-with-helm.md:470]:
```suggestion
|`prometheus.service.labels` | Kubernetes object labels to attach to the service object. | service: "nginx-ingress-prometheus-service" |
```

File: docs/content/installation/installation-with-helm.md [hunk 2]
```diff
@@ -338,10 +466,17 @@ The following tables lists the configurable parameters of the NGINX Ingress Cont
...
+|`prometheus.serviceMonitor.create` | Create a ServiceMonitor custom resource. Requires ServiceMonitor CRD to be installed. For the latest CRD, check the latest release on the [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator) GitHub repo under `example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml` | false |
+|`prometheus.serviceMonitor.labels` | Kubernetes object labels to attach to the serviceMonitor object. | "" |
+|`prometheus.serviceMonitor.selectorMatchLabels` | A set of labels to allow the selection of endpoints for the ServiceMonitor. | "" |
+|`prometheus.serviceMonitor.endpoints` | A list of endpoints allowed as part of this ServiceMonitor. | "" |
```
Comment (shaun-nx) [inline docs/content/installation/installation-with-helm.md:474]:
```suggestion
|`prometheus.serviceMonitor.labels` | Kubernetes object labels to attach to the serviceMonitor object. | {} |
|`prometheus.serviceMonitor.selectorMatchLabels` | A set of labels to allow the selection of endpoints for the ServiceMonitor. | service: "nginx-ingress-prometheus-service" |
|`prometheus.serviceMonitor.endpoints` | A list of endpoints allowed as part of this ServiceMonitor. | - port: prometheus |
```

File: charts/nginx-ingress/README.md [hunk 3]
```diff
@@ -190,6 +193,131 @@ The command removes all the Kubernetes components associated with the release an
...
+Although the advisory is to update all resources in accordance with new naming convention, to avoid the downtime
+please follow the steps listed in this page.
+
+### Upgrade Steps
+
+**Note** The following steps apply to both 2.x and 3.0.x releases.
+
+The steps you should follow depend on the Helm release name:
+
+{{<tabs>}}<!-- markdownlint-disable-line MD033 -->
```
Comment (lucacome) [inline charts/nginx-ingress/README.md:214]: we can't have this in the markdown

**Review by lucacome — CHANGES_REQUESTED** [review]

File: charts/nginx-ingress/README.md (hunk 3 above)
Comment (vepatel) [reply charts/nginx-ingress/README.md:214]: @lucacome this is blocking hugo custom tags, any other solution

Comment (lucacome) [reply charts/nginx-ingress/README.md:214]: don't use tabs? 😄

Comment (Jcahilltorre) [reply charts/nginx-ingress/README.md:214]: any other solution?

Comment (Jcahilltorre) [reply charts/nginx-ingress/README.md:214]: Also, can you elaborate further why we can't have this in the markdown? Might help us find a workaround.

Comment (lucacome) [reply charts/nginx-ingress/README.md:214]: I mean if you want to use tabs, no. Other than moving back to two separate documents and keeping them up to date manually.

Comment (Jcahilltorre) [reply charts/nginx-ingress/README.md:214]: to clarify, this is only an issue in the README, not in the content inside `/docs/` ?

Comment (ADubhlaoich) [reply charts/nginx-ingress/README.md:214]: The tab components should be removed from the README: they can instead be re-integrated as subheadings.

Comment (vepatel) [reply charts/nginx-ingress/README.md:214]: the issue is that README is source of truth for helm website docs and everything gets automatically overwritten during release, to make exceptions for <<tabs>>, we'll have to edit that release script I think .

**Pull request merged**